assets/src/dashboard/app/views/exploreTemplates/header/index.js

```diff
@@ -22,102 +22,38 @@ import { __ } from '@web-stories-wp/i18n';
 /**
  * External dependencies
  */
-import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { useFeature } from 'flagged';
 
 /**
  * Internal dependencies
  */
-import { Dropdown } from '../../../../components';
-import { DropdownContainer } from '../../../../components/dropdown';
 import {
   DASHBOARD_VIEWS,
   TEMPLATES_GALLERY_SORT_MENU_ITEMS,
-  DROPDOWN_TYPES,
 } from '../../../../constants';
 import {
   FilterPropTypes,
-  SearchPropTypes,
   SortPropTypes,
   ViewPropTypes,
 } from '../../../../utils/useTemplateView';
 import { useDashboardResultsLabel } from '../../../../utils';
-import { TemplatesPropType } from '../../../../types';
 import { PageHeading, BodyViewOptions } from '../../shared';
-import useTemplateFilters from '../templateFilters';
 
-const HeadingDropdownsContainer = styled.div`
-  display: flex;
-  align-items: baseline;
-  flex-direction: row;
-  justify-content: center;
-  width: 100%;
-
-  ${DropdownContainer} {
-    margin: 0 10px;
-    &:last-child {
-      margin-right: 0;
-    }
-  }
-`;
-function Header({ filter, totalTemplates, search, templates, sort, view }) {
+function Header({ filter, totalTemplates, sort, view }) {
   const enableInProgressTemplateActions = useFeature(
     'enableInProgressTemplateActions'
   );
 
   const resultsLabel = useDashboardResultsLabel({
-    isActiveSearch: Boolean(search.keyword),
     totalResults: totalTemplates,
     currentFilter: filter.value,
     view: DASHBOARD_VIEWS.TEMPLATES_GALLERY,
   });
 
-  const {
-    selectedCategories,
-    selectedColors,
-    onNewCategorySelected,
-    onNewColorSelected,
-    clearAllCategories,
-    clearAllColors,
-  } = useTemplateFilters();
-
-  const TemplateFilters = enableInProgressTemplateActions ? (
-    <HeadingDropdownsContainer>
-      <Dropdown
-        ariaLabel={__('Category Dropdown', 'web-stories')}
-        type={DROPDOWN_TYPES.PANEL}
-        placeholder={__('Category', 'web-stories')}
-        items={selectedCategories}
-        onClear={clearAllCategories}
-        onChange={onNewCategorySelected}
-      />
-      <Dropdown
-        ariaLabel={__('Color Dropdown', 'web-stories')}
-        type={DROPDOWN_TYPES.COLOR_PANEL}
-        placeholder={__('Color', 'web-stories')}
-        items={selectedColors}
-        onClear={clearAllColors}
-        onChange={onNewColorSelected}
-      />
-    </HeadingDropdownsContainer>
-  ) : null;
-
   return (
     <>
-      <PageHeading
-        centerContent
-        defaultTitle={__('Templates', 'web-stories')}
-        searchPlaceholder={__('Search Templates', 'web-stories')}
-        stories={templates}
-        showTypeahead={enableInProgressTemplateActions}
-        handleTypeaheadChange={
-          enableInProgressTemplateActions ? search.setKeyword : () => {}
-        }
-        typeaheadValue={search.keyword}
-      >
-        {TemplateFilters}
-      </PageHeading>
+      <PageHeading heading={__('Explore Templates', 'web-stories')} />
```

Comment:
Figma has heading of `Templates` updated to `Explore Templates` which makes sense because it matches the side nav.